navansatt/src/main/kotlin/no/nav/amt/tiltak/navansatt/NavAnsattDbo.kt

```diff
@@ -9,5 +9,4 @@ internal data class NavAnsattDbo(
 	val telefonnummer: String?,
 	val epost: String?,
 	val bucket: Bucket = Bucket.forNavIdent(navIdent)
```

Comment:
Mest for å lære, hva er dette?

Reply:
Hører til Einar sin PR fra en stund tilbake. Brukes for å splitte ansatte i buckets slik at når vi oppdaterer alle veilederene så får vi spredd lasten utover dagen.